byu-calendar-feature-column/byu-calendar-feature-column.scss

```diff
@@ -0,0 +1,100 @@
+$teal: #5d7998;
+$navy: #002e5d;
+$white: #fff;
+$gray: #666;
+
+#date, #date ::slotted(*) {
+  display: none;
+}
+
+:host {
+  width: 19%;
+  overflow: hidden;
+
+  .section-first {
+    width: 100%;
```

Comment:
Also go through and mixin-ify this file, and have a byu-calendar-feature-column-common.scss for mixins.